app/src/main/res/layout/view_decorated_text_item.xml

```diff
@@ -46,7 +46,8 @@
             style="@style/BodyMedium"
             android:layout_width="0dp"
             android:layout_height="match_parent"
-            android:layout_marginHorizontal="@dimen/marginStandardSmall"
+            android:layout_marginStart="@dimen/marginStandardSmall"
+            android:layout_marginEnd="@dimen/marginStandardMedium"
```

Comment:
This margin is not indicated in the specs, we need to request it because this is the only missing one